### PR TITLE
Replace TYPE with TYPES

### PR DESCRIPTION
**Pull request by kesselb**
As TYPE::* is deprecated.

### CODE REVIEW

**Review by ChristophWurst — APPROVED** [review]

**Comment by ChristophWurst** [comment]
I guess we should document this in the upgrade docs in case apps use it?

**Review by gary-kim — APPROVED** [review]
Yep, ran into this too

**Review by rullzer — CHANGES_REQUESTED** [review]
Block.

CI does :boom:

File: lib/private/DB/SQLiteMigrator.php [hunk 1]
```diff
@@ -29,7 +29,7 @@
 use Doctrine\DBAL\DBALException;
 use Doctrine\DBAL\Schema\Schema;
 use Doctrine\DBAL\Types\BigIntType;
-use Doctrine\DBAL\Types\Type;
+use Doctrine\DBAL\Types\Types as Type;
```
Comment (nickvergessen) [inline lib/private/DB/SQLiteMigrator.php:32]: Error: Call to undefined method Doctrine\DBAL\Types\Types::getType() in /drone/src/lib/private/DB/SQLiteMigrator.php:87

Comment (kesselb) [reply lib/private/DB/SQLiteMigrator.php:32]: Yep. I'm still undecided how to continue. The cleanest approach would be to not use an alias and migrate all references from `Type::BIGINT` (an others) to `Types::BIGINT`.

**Comment by kesselb** [comment]
Second try ;)

**Comment by gary-kim** [comment]
CI looks happy now

**Pull request merged**